packages/kbn-monaco/src/esql/language.ts

```diff
@@ -129,4 +129,36 @@ export const ESQLLang: CustomLangModuleType<ESQLCallbacks> = {
       },
     };
   },
+
+  getInlayHintsProvider: (callbacks?: ESQLCallbacks): monaco.languages.InlayHintsProvider => {
+    // @TODO: remove
+    console.log(`--@@monaco.languages.InlayHintKind.Type`, monaco.languages.InlayHintKind.Type);
+    return {
+      async provideInlayHints(
+        model: monaco.editor.ITextModel,
+        range: monaco.Range,
+        token: monaco.CancellationToken
+      ): Promise<ProviderResult<InlayHintList>> {
+        console.log(`--@@getInlayHintsProvider provideInlayHints`);
```

Comment:
But this line `console.log(`--@@getInlayHintsProvider provideInlayHints`);` is never triggered